### ISSUE TITLE
Installed modules translation error

### ISSUE THREAD

**Issue by md2158** [PrestaShop/PrestaShop#23720]
#### Describe the bug

I upgraded prestashop to 1.7.7.2 and can't translate installed modules. Getting this error:
'An error has occurred, this module does not exist'

How to solve this?

#### Additional information

* PrestaShop version: 1.7.7.2
* PHP version: 7.1

**Comment by hibatallahAouadni** [comment]
Hello @merick2158 

I can't reproduce your issue :confused: 
You need to mention the module name and the previous version of your shop.
And try to check if the module exist in the module list **Modules>Module Manager**.
May be your module is not compatible with PS1.7.7.2.
Please check and feedback.

PS: please edit your issue description and add the missing infos and blocks (steps to reproduce, screenshots).

Thanks!

**Comment by md2158** [comment]
Hi @hibatallahAouadni 
Upgraded from 1.7.3.4 to 1.7.7.2

Modules work fine, but all modules' translations can't be edited.

**Comment by hibatallahAouadni** [comment]
Hello @merick2158 

Could you please check this issue #13392 it seems similar to yours.
Please check and feedback.

Thanks!

**Comment by md2158** [comment]
Hi @hibatallahAouadni 

I checked the link but can't find any solution. Modules works, can't edit translations of all modules. Is there any solution?

**Comment by florine2623** [comment]
The issue #13392 is already in debug roadmap, yours is similar. in order to keep issue backlog clean, we close all issue duplicates.

In next version PS 1.7.8 it should be fixed : https://github.com/PrestaShop/PrestaShop/issues/13392#issuecomment-799521833

Thanks!

**Comment by florine2623** [comment]
Duplicate #13392

**Comment by md2158** [comment]
@florine2623 this is not the same issue, I can't even enter to translation page of installed modules. And all this happens after the upgrade. Your team always ignores bug issues, try to solve bugs guys.

**Comment by webmaxpro** [comment]
I had the same error. For some reason, the recaptcha and contactform modules were strangely interlinked. I managed to uninstall recaptcha. But when uninstalling contactform - there was an error "can't remove recaptcha module". Also when logging into the contactform configuration it took me to the recaptcha configuration page. Then managed to install updated contactform and after that reinstall recaptcha. After that, the error on the modules page and the error with no translations possible disappeared.

**Comment by kordama1** [comment]
Now I'm really worried..: > I had the same error. For some reason, the recaptcha and contactform modules were strangely interlinked. I managed to uninstall recaptcha. But when uninstalling contactform - there was an error "can't remove recaptcha module". Also when logging into the contactform configuration it took me to the recaptcha configuration page. Then managed to install updated contactform and after that reinstall recaptcha. After that, the error on the modules page and the error with no translations possible disappeared.< -> I desperately needed an answer for 'what EXACTLY should I perform, to get the free Google reCAPTCHA module installed.. within the e-commerce ([motohaft.pl](http://motohaft.pl/)), that I gathered, from PRESTA pages: https://prestapros.com/en/blog/free-module-recaptcha-for-prestashop-registration-form. Thanks in advance for your prompt answer! The quuestion was just reg. WHERE to I should paste those 4 zipped files into???
